.github/workflows/audit.yaml

```diff
@@ -1,11 +1,11 @@
 name: Security audit
 on:
   schedule:
-    - cron: '0 0 * * *'
+    - cron: "0 0 * * *"
   workflow_dispatch:
   pull_request:
 env:
-  rust_version: 1.51.0
+  rust_version: 1.54.0
```

Comment:
Can we add a note to the release guide once we ship this? I don't want us to forget to update this every 2 weeks